src/api.js

```diff
@@ -11,14 +11,12 @@ export const getArticles = () => {
         };
 
 
-// export const getBasket = () => {
-//   return api.get(`/users/Ant/basket`).then(({ data }) => {
-//     return data;
-//   });
-// };
 
-// export const deleteItemFromBasket = (item_id) => {
-//   return api.delete(`/users/Ant/basket/${item_id}`).then(({ data }) => {
-//     return data;
-//   });
-// };
+export const getArticleById = (article_id) => {
+    return api.get(`/articles/${article_id}`).then(({ data }) => {
+            return data;
+            });
+          };
```

Comment:
Should we just be sending back the data object, or is there some more specific this should return within the data object?